packages/amplify-cli/src/extensions/amplify-helpers/resource-status.ts

```diff
@@ -9,13 +9,17 @@ import { CLOUD_INITIALIZED, CLOUD_NOT_INITIALIZED, getCloudInitStatus } from './
 import { ServiceName as FunctionServiceName, hashLayerResource } from 'amplify-category-function';
 import { removeGetUserEndpoints } from '../amplify-helpers/remove-pinpoint-policy';
 import { pathManager, stateManager, $TSMeta, $TSAny, NotInitializedError } from 'amplify-cli-core';
+import { ViewResourceTableParams } from "amplify-cli-core/lib/cliViewAPI";
```

Comment:
Can we export  ```ViewResourceTableParams``` index file so that we can directly import from 'amplify-cli-core'

Reply:
will do